### ISSUE TITLE
Weight Initialisation

### ISSUE THREAD

**Issue by RiboRings** [irenebernardi/VGG-16-replication#4]
We would like to initialise weights from normal distribution with params given in the paper, and make it reproducible (every time it runs it should give identical results, ideally using same seed as authors). Check [keras normal initializer](https://keras.io/api/layers/initializers/). Once the initialiser is in place, set seed and make sure you get same results every time you build and train model.
